src/Drawer/Drawer.spec.js

```diff
@@ -1,7 +1,7 @@
 /* eslint-env mocha */
 import React from 'react';
 import {shallow} from 'enzyme';
-import sinon from 'sinon';
+import consoleError from '../../test/utils/consoleError';
```

Comment:
Sorry, I would have provided more detail. I was suggesting it injecting it in the [setup](https://github.com/callemall/material-ui/blob/master/test/index.js) so every tests benefits from it.